app/controllers/mixins/ems_common_angular.rb

```diff
@@ -152,6 +152,14 @@ def get_task_args(ems)
           :metrics_port     => params[:metrics_api_port],
           :metrics_database => params[:metrics_database_name],
         }]
+      when 'ManageIQ::Providers::Kubevirt::InfraManager'
+        [{
+          :token      => params[:default_kubevirt_token],
```

Comment:
Out of curiosity, why does the parameter have the word kubevirt in it (as opposed to `default_token` or `default_provider_token`.  We should strive for pluggability and less hardcoding where possible...obviously the UI is not pluggable, but adding provider names to key names seems a step in the wrong direction.